src/base/17.0.1.3/attr_domains2expr.py

```diff
@@ -320,8 +320,13 @@ def fix_elem(cr, model, elem, comb_arch):
             )
         for key in extra:
             value = ast.unparse(attrs[key])
-            _logger.info("Inlined %s=%r", key, value)
-            elem.set(key, value)
+            try:
+                etree.QName(key)
+            except ValueError:
+                _logger.exception("Skipping invalid attribute name %r in `attrs` with value: %s", key, value)
```

Comment:
We don't need the traceback.
```suggestion
            except ValueError as e:
                _logger.error("Skipping invalid attribute name %r in `attrs` with value: %s: %r", key, value, e)  # noqa: TRY400
```

Reply:
Done :).